src/server/enrichment-map/emap/generateCys.js

```diff
@@ -20,7 +20,8 @@ const _ = require('lodash');
 
 // input ["GO:1902275", "GO:2001252", "GO:1905269", "GO:0051053"]
 // returns a cytoscape object
-const generateCys = (pathwayIdList) => {
+const generateCys = (pathwayIdList, cutoff = 0.375) => {
+  if (cutoff < 0 || cutoff > 1) { throw 'ERROR: cutoff out of range [0,1]';}
```

Comment:
Do not throw `msg` strings.  Throw `new Error(msg)` objects: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error